README.md

```diff
@@ -104,15 +66,24 @@ annotation. This is equivalent to the `test-network-function.com/container_tests
 * `test-network-function.com/subscription_name` is optional and should contain a JSON-encoded string that's the name of
 the subscription for this CSV. If unset, the CSV name will be used.
 
-### Runtime environement variables to skip tests during development
-#### Turn off openshift required tests when CNF run on kubernetes only environment
-when test on CNFs that run on k8s only environment, execute shell command below before compile tool and run test shell script.
+### testPartner
```

Comment:
### testPartner

This section can also be discovered automatically and should be left commented out unless the parter pods are modified from the original version in [cnf-certification-test-partner](https://github.com/test-network-function/cnf-certification-test-partner/local-test-infra/)

#### partnerContainers
The partner container is used as a partner to initialte connectivity tests
##### Configuration via config file
```shell script
testPartner:
  partnerContainers: # partner containers are listed under this
    - namespace: tnf # the namespace of the partner pod
      podName: partner # the name of the partner pod
      containerName: partner # the container name of the partner pod's container to use
      defaultNetworkDevice: eth0 # the default kubernetes core interface in the pod (Not multus)
      multusIpAddresses: # the multus IP address
        - 10.217.0.29
```

##### Auto-discovery with labels
For auto-configuration, do the add the following label to the pod:
```shell script
test-network-function.com/generic: orchestrator	
```

#### fsDiffMasterContainer
The FSDiffMasterContainer is used as a reference to compare against any modifications in the pod under test during runtime
##### Configuration via config file

```shell script
testPartner:
  fsDiffMasterContainer:
    namespace: tnf # the namespace of the partner pod
    podName: node-master # the name of the pod used to test filesystem
    containerName: master # the name of the container to use in the pod   
```
##### Auto-discovery with labels
For auto-configuration, do the add the following label to the pod:
```shell script
test-network-function.com/generic=fs_diff_master
```



### certifiedcontainerinfo and certifiedoperatorinfo